WorkAndFun/src/main/resources/templates/announcementsList.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+          integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link th:href="@{/css/style.css}" rel="stylesheet"/>
+
+    <title>Wszystkie ogłoszenia</title>
+</head>
+
+
+<body class="bg-light p-3">
+<div class="container">
+    <div class="row">
+        <div class="announcement-frame d-block bg-white form-group col-12" th:each="announcement, stat : ${announcements}">
+            <div class="row">
+                <div class="announcement-left-side p-3 col-8 col-md-10">
+                    <h4 class="announcement-header" th:text="${announcement.getHeader()}"></h4>
+                    <p class="announcement-date-and-location"
```

Comment:
Opakowane w diva działa jak należy, responsywnie. 
`<div class="position-absolute bottom-0"><p
                       th:text="${announcement.getFullLocalization() + ' - ' + announcement.announcementCreationDateFormatted()}"></p></div>
                </div>`

Reply:
@BienkowskiTomasz przerobiłem troszkę, bo Paweł mi pomógł - teraz nie wyjeżdża mi data poniżej ogłoszenia. Ale jak dodałem ten `position-absolute bottom-0` to właśnie ucieka mi data pod ogłoszenie. Jak zpuszuję poprawki to dam znac żebyś sprawdził, bo moim zdaniem i tak będzie dobrze